0016/endpoint-time-series-event.avsc

```diff
@@ -0,0 +1,102 @@
+{
+    "namespace":"org.kaaproject.ipc.collector2ts.gen.v1",
+    "name":"TimeSeriesEvent",
+    "type":"record",
+    "fields":[
+        {
+            "name":"correlationId",
+            "type":"string",
+            "doc":"Message ID primarily used to track message processing across services"
+        },
+        {
+            "name":"timestamp",
+            "type":"long",
+            "doc":"Message creation UNIX timestamp in milliseconds"
+        },
+        {
+            "name":"timeout",
+            "type":"long",
+            "default":0,
+            "doc":"Amount of milliseconds since the timestamp until the message expires. Value of 0 is reserved to indicate no expiration."
+        },
+        {
+            "name":"appVersionName",
+            "type":"string",
+            "doc":"Application version name the data is sent for"
```

Comment:
Not sure `appVersionName` is of any use. Data points do not belong to any appversion anyway.